docs/en/docs/sql-manual/sql-functions/array-functions/sequence.md

```diff
@@ -0,0 +1,107 @@
+---
+{
+    "title": "SEQUENCE",
+    "language": "en"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## sequence
+
+<version since="dev">
+
+sequence
+
+</version>
+
+### description
+alias of array_range function
```

Comment:
if it is only a alias, i think we'd better use one doc for good maintainability

Reply:
But without the doc of sequence, how can we tell the users the way to search and use this quiet common function?